pittgoogle/schemas/lsst/8/0/lsst.v8_0.MPCORB.avsc

```diff
@@ -0,0 +1,115 @@
+{
+  "type": "record",
+  "namespace": "lsst.v8_0",
+  "name": "MPCORB",
+  "fields": [
+    {
+      "default": null,
+      "doc": "Number or provisional designation (in packed form).",
+      "name": "mpcDesignation",
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    {
+      "default": null,
+      "doc": "LSST unique identifier (if observed by LSST).",
+      "name": "ssObjectId",
+      "type": [
+        "null",
+        "long"
+      ]
+    },
+    {
+      "default": null,
+      "doc": "Absolute magnitude, H.",
+      "name": "mpcH",
+      "type": [
+        "null",
+        "float"
+      ]
+    },
+    {
+      "default": null,
+      "doc": "Epoch (in MJD, .0 TT).",
+      "name": "epoch",
+      "type": [
+        "null",
+        "double"
+      ]
+    },
+    {
+      "default": null,
+      "doc": "Mean anomaly at the epoch, in degrees.",
+      "name": "M",
+      "type": [
+        "null",
+        "double"
+      ]
+    },
+    {
+      "default": null,
+      "doc": "Argument of perihelion, J2000.0 (degrees).",
+      "name": "peri",
+      "type": [
+        "null",
+        "double"
+      ]
+    },
+    {
+      "default": null,
+      "doc": "Longitude of the ascending node, J2000.0 (degrees).",
+      "name": "node",
+      "type": [
+        "null",
+        "double"
+      ]
+    },
+    {
+      "default": null,
+      "doc": "Inclination to the ecliptic, J2000.0 (degrees).",
+      "name": "incl",
+      "type": [
+        "null",
+        "double"
+      ]
+    },
+    {
+      "default": null,
+      "doc": "Orbital eccentricity.",
+      "name": "e",
+      "type": [
+        "null",
+        "double"
+      ]
+    },
+    {
+      "default": null,
+      "doc": "Semimajor axis (AU).",
+      "name": "a",
+      "type": [
+        "null",
+        "double"
+      ]
+    },
+    {
+      "default": null,
+      "doc": "Perihelion distance (AU).",
+      "name": "q",
+      "type": [
+        "null",
+        "double"
+      ]
+    },
+    {
+      "default": null,
+      "doc": "MJD of pericentric passage.",
+      "name": "t_p",
+      "type": [
+        "null",
+        "double"
+      ]
+    }
+  ]
+}
```

Comment:
We should probably go ahead and add a blank line. It's basically a universal standard to end all files with a single blank line (afaik, POSIX systems require this in order for them to read the file correctly). Looks like lsst.v8_0.alert.avsc already has this, so that reassures me that adding one to the others won't mess anything up.

```suggestion
}

```